network/src/connectivity_manager/mod.rs

```diff
@@ -105,16 +113,16 @@ pub enum ConnectivityRequest {
     /// Update set of nodes eligible to join the network.
     UpdateEligibleNodes(HashMap<PeerId, x25519::PublicKey>),
     /// Gets current size of dial queue. This is useful in tests.
-    #[serde(skip)]
-    GetDialQueueSize(oneshot::Sender<usize>),
+    GetDialQueueSize(#[serde(skip)] oneshot::Sender<usize>),
```

Comment:
What's up yo?

Reply:
Oh, `serde` was doing this weird thing complaining that oneshot wasn't serializable that wasn't happening to me before.  When I moved it to just be on the `sender` it stopped complaining, which is strange because if I skipped the whole field, why would it care about the subfield.

This happened in the context of structured logging.  Let me check if it's still happening